etils/epath/abstract_path.py

```diff
@@ -19,7 +19,7 @@
 import os
```

Comment:
Please also update the `CHANGELOG.md` file

Reply:
done